### PR TITLE
Imx6ull secondary boot (part 2)

### PR DESCRIPTION
**Pull request by igoropaniuk**
Please do not submit a Pull Request via github.  Our project makes use of
mailing lists for patch submission and review.  For more details please
see https://www.denx.de/wiki/U-Boot/Patches

### CODE REVIEW

File: arch/arm/mach-imx/imx_secondary_boot.c [hunk 1]
```diff
@@ -25,10 +25,15 @@ static int do_imx_secondary_boot(cmd_tbl_t *cmdtp, int flag,
 
 	/* If we just want to retrieve current value */
 	if (argc == 1) {
-		arm_smccc_smc(IMX_SIP_SRC, IMX_SIP_SRC_IS_SECONDARY_BOOT, 0, 0,
-			      0, 0, 0, 0, &res);
+		if (CONFIG_IS_ENABLED(ARCH_IMX8M)) {
+			arm_smccc_smc(IMX_SIP_SRC,
+				      IMX_SIP_SRC_IS_SECONDARY_BOOT, 0, 0,
+				      0, 0, 0, 0, &res);
```
Comment (ricardosalveti) [inline arch/arm/mach-imx/imx_secondary_boot.c:31]: On the imx6 case res will be defined but not used, won't it cause a warning?

**Review by ricardosalveti — COMMENTED** [review]
LGTM, one minor comment.

File: arch/arm/mach-imx/imx_secondary_boot.c (hunk 1 above)
Comment (igoropaniuk) [reply arch/arm/mach-imx/imx_secondary_boot.c:31]: Yes, will fix

**Review by vanmaegima — APPROVED** [review]

**Review by mike-scott — APPROVED** [review]
LGTM

**Pull request merged**